source/includes/read/count.cpp

```diff
@@ -0,0 +1,64 @@
+#include <iostream>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main() {
+    mongocxx::instance instance{};
+    mongocxx::uri uri("<connection string>");
+    mongocxx::client client(uri);
+
+    // start-db-coll
+    auto db = client["sample_training"];
+    auto collection = db["companies"];
+    // end-db-coll
+ 
+    {
+        // Counts all documents in the collection
+        // start-count-all
+        auto result = collection.count_documents({});
+        std::cout << "Number of documents: " << result << std::endl;
+        // end-count-all
+    }
+
+    {
+        // Counts documents that have a "founded_year" value of 2010
+        // start-count-accurate
+        auto result = collection.count_documents(make_document(kvp("founded_year", 2010)));
+        std::cout << "Number of companies founded in 2010: " << result << std::endl;
+        // end-count-accurate
+    }
+
+    {
+        // Counts a maximum of 100 documents that have a "number_of_employees" value of 50
+        // start-modify-accurate
+        mongocxx::options::count opts;
+        opts.limit(100); 
+        auto result = collection.count_documents(make_document(kvp("number_of_employees", 50)));
```

Comment:
```suggestion
        auto result = collection.count_documents(make_document(kvp("number_of_employees", 50)), opts);
```
Missing `opts` argument specifying the limit set above.